src/core/vectortile/qgsmapboxglstyleconverter.cpp

```diff
@@ -281,6 +283,7 @@ bool QgsMapBoxGlStyleConverter::parseFillLayer( const QVariantMap &jsonLayer, Qg
         if ( ddProperties.isActive( QgsSymbolLayer::PropertyFillColor ) )
         {
           context.pushWarning( QObject::tr( "%1: Could not set opacity of layer, opacity already defined in fill color" ).arg( context.layerId() ) );
```

Comment:
These warnings can be removed now, right?

Reply:
Yes, they are obsolete now